### ISSUE TITLE
[Bug] The learning information of the home page is blocked by the rounded border of the screen.

### ISSUE THREAD

**Issue by gugutu** [ankidroid/Anki-Android#14660]
The learning information of the home page is blocked by the rounded border of the screen.

![1699086141144.jpg](https://github.com/ankidroid/Anki-Android/assets/59798823/713bdfff-6c78-48f2-b973-36b506543413)

Ankidroid version: 2.17 alpha 3

**Comment by snowtimeglass** [comment]
Might be a duplicate of #14641.

**Comment by david-allison** [comment]
Closing as duplicate
